student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(age, classroom)
+    super(age)
+    @classroom = classroom
+  end
```

Comment:
- [ ] In the `initialize` method of the `Student` class, you need to pass in the `name` and `parent_permission` data as parameters. Since the `Student` class is inherited from the `Person` class, you need to pass in the `name` and `parent_permission` data as parameters of the `super` method. 👍 😎 

#### Hint:

```ruby
def initialize(age, classroom, name = 'Unknown', parent_permission = true)
  super(age, name, parent_permission: parent_permission)
  @classroom = classroom
end
```